common/short-hex-str/src/lib.rs

```diff
@@ -16,8 +16,10 @@ use thiserror::Error;
 pub struct ShortHexStr([u8; ShortHexStr::LENGTH]);
 
 #[derive(Error, Debug)]
-#[error("Input bytes are too short")]
-pub struct InputTooShortError;
+pub enum Error {
+    #[error("Input bytes are too short")]
+    InputTooShortError,
+}
```

Comment:
Can you revert this change since you no longer have another error variant?

Reply:
Sure, reverting it rn